### DISCUSSION TITLE
`Copyq toggle` not showing up in the tag that it is called from

### DISCUSSION THREAD

**Discussion by RayZ0rr** [herbstluftwm/herbstluftwm#1552] [Q&A]
I am using [CopyQ](https://github.com/hluk/CopyQ) and have these in my autostart:

```bash
copyq &
hc keybind $Mod-q	      spawn copyq toggle
```

However, once I press `MOD+q` in a particular tag and copyq appears, I hide it and switch to another tag and then press it again, it appears in the initial tag that it first appeared.

I raised an [issue there](https://github.com/hluk/CopyQ/issues/2064) and the dev said it's not reproducible. 

Could anyone see if this is reproducible as I am still having this issue.

**Comment by t-wissmann** [comment]
Hi, there is definitely something odd going on which might be hlwm's fault. Here, it works well if I hide the window (via the keybinding) before going to the other tag. However, if I first go to the other tag, I can't get copyq show up, regardless of how often I press the keybinding. Unfortunately, I currently don't have the time to investigate it further

**Comment by RayZ0rr** [comment]
Hi @t-wissmann , thank you for testing this.

I can't get copyq to show even if I hide it before switching tags.

Can you share your autostart or related settings?

BTW the behaviour you described is absolutely fine for almost all usecases.

There's also `copyq menu` ( `hc keybind $Mod-q	      spawn copyq menu` ) which can show even if tag switching is done first.

**Reply by t-wissmann** [reply]
You can find my settings in [my dotfiles](https://github.com/t-wissmann/dotfiles), but I would assume that it should not have much of an effect.

**Reply by RayZ0rr** [reply]
I looked at your config.

I didn't find ` ~/git/barpyrus/barpyrus.py ` file

I can reproduce your behaviour when I have stalone tray present. When stalonetray is closed I can't get that behaviour

**Reply by t-wissmann** [reply]
Barpyrus is in its own repository: https://github.com/t-wissmann/barpyrus

In my barpyrus config, I have a tray (via stalonetray), and this could make a difference indeed!

**Reply by RayZ0rr** [reply]
When you have time, today, next month, 3 months later etc, can u test this with/without stalonetray enabled?

**Reply by t-wissmann** [reply]
I hope I can take a look at it next week.

**Comment by ArijanJ** [comment]
This is not a perfect solution but I'm using the same software and I have a scratchpad using a modified scratchpad script from the repo, I have a `hc rule class="copyq" tag=SCR` and I run `copyq & copyq show` in my autostart, then instead of toggling copyq I toggle the scratchpad and do my copyq work there. Just in case you're looking for a usable solution.

**Reply by RayZ0rr** [reply]
Thanks for the tip. Right now I am managing with `copyq menu` as it works as expected.
